cmds/core/cpio/cpio_test.go

```diff
@@ -140,24 +159,31 @@ func TestCpio(t *testing.T) {
 }
 
 func TestDirectoryHardLink(t *testing.T) {
-	tempDir := t.TempDir()
 
 	// Open an archive containing two directories with the same inode (0).
 	// We're trying to test if having the same inode will trigger a hard link.
 	archiveFile, err := os.Open("testdata/dir-hard-link.cpio")
 	if err != nil {
 		t.Fatal(err)
 	}
-	c := testutil.Command(t, "-v", "i")
-	c.Dir = tempDir
-	c.Stdin = archiveFile
 
-	out, err := c.Output()
+	// Change directory back afterwards to not interfer with the subsequent tests
+	wd, err := os.Getwd()
 	if err != nil {
-		t.Fatalf("Extraction failed:\n%s\n%s\n%v\n", out, c.Stderr, err)
+		t.Fatalf("Could not get current working directory: %v", err)
+	}
+	defer os.Chdir(wd)
+	tempExtractDir := os.TempDir()
```

Comment:
Using `t.TempDir()` might be preferable here (https://pkg.go.dev/testing#B.TempDir) because it will clean up after your test :)

Reply:
good point!

Reply:
oh, also, me sure to do a git pull --rebase before you submit again ... there's a whole bunch of commits in here that don't relate to this PR.

Reply:
Thanks for the suggestion. I cleaned up the testing as suggested and rebased. Sorry that the history got a bit messy.